OpenXmlPowerTools/ColorParser.cs

```diff
@@ -29,14 +29,30 @@ public static Color FromName(string name)
 
         public static bool TryFromName(string name, out Color color)
         {
-            color = Color.FromName(name);
-
-            return color.IsNamedColor;
+            try
+            {
+                color = Color.FromName(name);
+
+                return color.IsNamedColor;
+            }
+            catch
+            {
+                color = new Color();
```

Comment:
Does this need to be set to an instance? Why not set it to null? If it's a struct, use `default(Color)`